docs/reference/chatbot-ollama.md

```diff
@@ -0,0 +1,141 @@
+# chatbot-ollama
+
+- [:material-account-group: Main author - HE-Arc](https://www.hes-so.ch/swiss-ai-center/equipe)
+- [:material-git: Code](https://github.com/swiss-ai-center/chatbot-ollama-service)
+- [:material-kubernetes: Deployment configuration](https://github.com/swiss-ai-center/chatbot-ollama-service/tree/main/kubernetes)
+- [:material-test-tube: Staging](https://chatbot-ollama-swiss-ai-center.kube.isc.heia-fr.ch/)
+- [:material-factory: Production (not available yet)](https://chatbot-ollama.swiss-ai-center.ch/)
+
+This service is a streamlit interface to interact with a large language model
+using the [ollama API](https://ollama.ai/) and langchain.
+
+Ollama is an open source project that aims to provide a simple and easy to use
+app to download and use LLM models like llama2 or mistral locally.
+
+On the [Ollama official website](https://ollama.ai/), you can find the list of
+available models and the link to the
+[github repository](https://github.com/jmorganca/ollama).
+
+Before using this service, you need to vectorize a pdf document using the
+[document-vectorizer service](/reference/services/document-vectorizer/).
+
+## Environment variables
+
+All environment variables are described in the
+[`.env`](https://github.com/swiss-ai-center/chatbot-ollama-service/blob/main/.env)
+file.
+
+The environment variables can be overwritten during the CI/CD pipeline described
+in the
+[`workflow.yml`](https://github.com/swiss-ai-center/chatbot-ollama-service/blob/main/.github/workflows/workflow.yml)
+GitHub workflow file.
+
+## Start the service locally with Python
+
+In the `services/chatbot-ollama` directory, start the service with the following
```

Comment:
I guess this should be the `chatbot-ollama` directory instead of `services/chatbot-ollama`?